lms/templates/dashboard.html

```diff
@@ -63,13 +63,13 @@
 <div class="dashboard-notifications" tabindex="-1">
     %if message:
         <section class="dashboard-banner">
-            ${message}
+            ${message | n, unicode}
```

Comment:
Nit (and I know it is too late): I would have recommended naming these variables something with HTML in it so the special handling is clear. Like message_html and enrollment_message_html.